packages/create-vite/template-solid-ts/src/index.tsx

```diff
@@ -0,0 +1,15 @@
+/* @refresh reload */
+import { render } from 'solid-js/web'
+
+import './index.css'
+import App from './App'
+
+const root = document.getElementById('root')
+
+if (import.meta.env.DEV && !(root instanceof HTMLElement)) {
+  throw new Error(
+    'Root element not found. Did you forget to add it to your index.html? Or maybe the id attribute got mispelled?',
+  )
+}
+
+render(() => <App />, root!)
```

Comment:
I think we should directly render it like the other templates

Reply:
So just remove the error throw?

Reply:
Yep